### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by musangamfure**
**Hello Reviewer 👋,**

In this PR, I have worked on adding a basic UI feature to the school library app project.

## **Here is what I have done:**

- [ ] Create an `app.rb` file, which will serve as my console app entry-point.
- [ ] Define methods in `app.rb` for listing all books, listing all people, creating a person (teacher or student), creating a book, creating a rental, and listing all rentals for a given person's id.
- [ ] create a `main.rb` file, and define the entry point method main, which will be invoked at the end of the file. This method presents the user with a list of options, let the user choose an option, ask for parameters if needed, and provide a way to quit the app

Kindly check my code and let me know what you think.

**Best Regards.**

Happy coding🎉🎉

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    3 => CreateStudentHandler.new,
+    4 => CreateTeacherHandler.new,
+    5 => CreateBookHandler.new,
+    6 => CreateRentalHandler.new,
+    7 => ListRentalsForPersonHandler.new
+  }
+
+  handler = option_handler[choice]
+  handler&.handle || puts('Invalid option, please try again.')
+end
```
Comment (cynthiainga) [inline main.rb]: - [ ] Please kindly make sure your console app can be running without any problem. When I run the app and chose any option in the menu I get an error, please kindly fix that 🙏🏾 🙏🏾 

Screenshot 2023-07-27 at 14 18 05

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @musangamfure, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Descriptive PR ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @musangamfure,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by musangamfure** [comment]
Thanks, @DammyShittu for approving my project🙏💪

**Pull request merged**